extensions/panache/mongodb-panache/runtime/src/test/java/io/quarkus/mongodb/panache/runtime/MongoOperationsTest.java

```diff
@@ -273,4 +352,43 @@ public void testBindUpdate() {
                 Parameters.with("field", "a value").map());
         assertEquals("{'$set':{'field':'a value'}}", update);
     }
+
+    //    @Test
+    //    public void testBuildNativeQueryWithInOperator() {
+    //        List<Object> list = Arrays.asList("id1", "id2");
+    //        String query = MongoOperations.bindFilter(DemoObj.class, "{ id: { $in: [?1] } }", new Object[]{list});
+    //        assertEquals("{ id: { $in: ['id1', 'id2'] } }", query);
+    //    }
+    //
+    //    @Test
+    //    public void testBuildNativeWithInPlusAndOperators() {
+    //        List<Object> list = Arrays.asList("id1", "id2");
+    //        String query = MongoOperations.bindQuery(DemoObj.class, "{ id: { $in: [?1] }, isOk: ?2 }", new Object[]{list, true});
+    //        assertEquals("{ id: { $in: ['id1', 'id2'] }, isOk: true }", query);
+    //    }
+    //
+    //    @Test
+    //    public void testBuildNativeWithInPlusOrOperators() {
+    //        List<Object> list = Arrays.asList("id1", "id2");
+    //        String query = MongoOperations.bindQuery(DemoObj.class,
+    //                "{ id: { $in: [?1] }, $or: [ {'property': ?2}, {'property': ?3} ] }",
+    //                new Object[]{list, "jpg", "gif"});
+    //        assertEquals("{ id: { $in: ['id1', 'id2'] }, $or: [ {'property': 'jpg'}, {'property': 'gif'} ] }", query);
+    //    }
+    //
+    //    @Test
+    //    public void testBuildNativeWithInPlusAndPlusOrOperators() {
+    //        List<Object> list = Arrays.asList("id1", "id2");
+    //        String query = MongoOperations.bindQuery(DemoObj.class,
+    //                "{ _id: { $in: [?1] }, isOk: ?2, $or: [ {'property': ?3}, {'property': ?4} ] }",
+    //                new Object[]{list, true, "jpg", "gif"});
+    //        assertEquals("{ _id: { $in: ['id1', 'id2'] }, isOk: true, $or: [ {'property': 'jpg'}, {'property': 'gif'} ] }", query);
+    //    }
+    //
+    //    @Test
+    //    public void testBuildQueryWithInOperator() {
+    //        List<Object> list = Arrays.asList("id1", "id2");
+    //        String query = MongoOperations.bindQuery(DemoObj.class, "id in ?1", new Object[]{list});
+    //        assertEquals("{ id: { $in: ['id1', 'id2'] } }", query);
+    //    }
```

Comment:
Don't commit commented code.
Remove it or uncomment it.

Reply:
Sorry man, I hate comments in the code also.